handlers/bot_handler.py

```diff
@@ -54,14 +54,36 @@ def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_ad
             raise InvalidCommand("Sorry, couldn't retrieve the git information for the bot...")
 
 
+class InviteCommand(Command):
+    """
+    Invite a list of members to the current channel, parses out members already
+    present
+    """
+
+    @classmethod
+    def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
+        """Execute the Invite command"""
+        try:
+            current_members = slack_wrapper.get_channel_members(channel_id)
+            # strip uid formatting
+            invited_users = [user.strip("<>@") for user in args]
```

Comment:
Will this work if i list users without the @? that would be nice.

Reply:
it doesn't but it can

would need to get info on all users and search through for uids for each username. i can add in a separate pr if you want

Reply:
So when I use @grazfather, the slack bot gets the UID directly? I understand why we might want to avoid a big api call and search.

Reply:
yeah, this is what the debug log looks like from the screenshot i included above:

```
DEBUG:log:Processing message: invite <@UG1R7DGQY> <@UG2L117AP> from CG464PA3G (UG1R7DGQY)
2019-02-09 17:31:29,456 - botserver            - Received bot command : version (CG464PA3G)
```

slack internally resolves `@` into unique uids for users. fun fact, slackbot's uid is `<@USLACKBOT>`